CompetitionTemplate/src/competition/subsystems/hanger/hook_commands/HookExtendCommand.java

```diff
@@ -0,0 +1,35 @@
+package competition.subsystems.hanger.hook_commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.hanger.HookSubsystem;
+import xbot.common.command.BaseCommand;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class HookExtendCommand extends BaseCommand{
+    public HookSubsystem hookSubsystem;
+    DoubleProperty hookExtentionPower;
+    
+    @Inject
+    public HookExtendCommand(HookSubsystem hookSubsystem, XPropertyManager propManager){
+        this.hookSubsystem = hookSubsystem;
+        hookExtentionPower = propManager.createPersistentProperty("hook extention power", -1.0);
```

Comment:
To be in keeping with the rest of the robot - positive power should go up, and negative power should go down.